### PR TITLE
Fix/binaries import into ios bundle

### PR DESCRIPTION
**Pull request by siepra**

### CODE REVIEW

File: packages/mobile/nodejs-assets/override-dlopen-paths-data.json [hunk 1]
```diff
@@ -0,0 +1 @@
+[{"originalpath":["node_modules","classic-level","build","Release","classic_level.node"],"newpath":["..","Frameworks","classic-level.framework","classic-level"]}]
```
Comment (kowalski) [inline packages/mobile/nodejs-assets/override-dlopen-paths-data.json:1]: weird you don't have and entry for one for leveldown.node

Comment (siepra) [reply packages/mobile/nodejs-assets/override-dlopen-paths-data.json:1]: huh of course you're right, but I saw it working on my iPhone

Comment (kowalski) [reply packages/mobile/nodejs-assets/override-dlopen-paths-data.json:1]: :shrug:

**Review by kowalski — APPROVED** [review]

File: packages/mobile/nodejs-assets/override-dlopen-paths-data.json (hunk 1 above)
Comment (siepra) [reply packages/mobile/nodejs-assets/override-dlopen-paths-data.json:1]: Ok, it looks like the app doesn't care about the path for leveldown (strange). I'll commit the file containing proper mapping for it anyway

**Comment by siepra** [comment]
Mobile applications won't work atm because `importDynamically` method is not adjusted for android/iOS

**Pull request merged**